account_storno/README.rst

```diff
@@ -0,0 +1,89 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+
+==============
+Account Storno
+==============
+
+This module adds the posibility to use Storno Accounting, a business practice
+commonly used in Eastern European countries.
+
+Storno accounting is a practice of using negative debit or credit amounts
+to reverse original journal account entries.
+Because bookkeepers typically write Storno entries in red ink,
+this accounting practice is also known as Red Storno.
+
+There is a difference between debit turnover and credit turnover,
+because the reverse entry makes redundant debit and credit turnover.
+
+
+Countries where Storno accounting is mandatory or considered as best practice:
+
+* Bosnia and Herzegovina
+* China
+* Czech Republic
+* Croatia
+* Poland
+* Romania
+* Russia
+* Serbia
+* Slovakia
+* Slovenia
+* Ukraine
+
+Configuration
+=============
+
+To configure this module:
+
+* Go to the menu Accounting/Invoicing > Configuration > Accounting > Journals
+* Open in form view the journals for which you want to allow storno 
+* Change the value of field Posting policy to "Storno"
```

Comment:
@feketemihai Hi,

As this is a Country/Company-wide setting it would be better to have it set in the company as AngloSaxon Setting is set right now.

Regards

Reply:
@hbto I don't think so, because you can still have an "AngloSaxon" accounting and in the same time to use storno, Storno Accounting is more about procedure of refund a invoice, that's why we leave the option to journal, because bank/cash journals should not apply this option, you book just like in the standard Odoo accounting.

Reply:
@elicoidal @hbto In the next period i will supply new modules, like account_voucher_storno, stock_account_storno, hr_expense_storno...

Reply:
Agree with @feketemihai current setup is multi-company compatible (at journal level)